src/app/models/Log.tsx

```diff
@@ -21,98 +21,53 @@ export interface Log {
 }
 
 const LogModel = {
-    list: [] as any[],
-    current: {} as Log,
-    createLog: {} as Log, // log being created
-    async fetch(query?: string) {
-        return m.request({
-            method: 'GET',
-            url: `${API_URL}logs${query ? `?${query}` : ''}`,
-            withCredentials: false
-        }).then((result: any) => {
-            this.list = result;
-        }).catch(() => {
-            const result = dummyLogList;
-            this.list = result;
-        });
+    list: {
```

Comment:
Maybe this should have a different name, one that's more obvious?

Reply:
What should have a different name?